bindings/go/constructor/go.mod

```diff
@@ -18,6 +18,10 @@ require (
 	sigs.k8s.io/yaml v1.6.0
 )
 
+replace (
+	ocm.software/open-component-model/bindings/go/dag => ../dag
```

Comment:
until https://github.com/open-component-model/open-component-model/pull/1106 is merged